server/app.py

```diff
@@ -146,7 +154,7 @@ def get_schools():
     Session, engine = dbconnect(db_options)
     session = Session()
     schools = session.query(School).all()
-    print schools
+    print (schools)
```

Comment:
python 3

Reply:
we shuoldn't be printing anyway, delete all print statements. we can setup logging functionality for that

Reply:
http://flask.pocoo.org/docs/1.0/logging/